woocommerce-payments.php

```diff
@@ -150,6 +149,7 @@ function () {
  */
 function wcpay_init() {
 	require_once WCPAY_ABSPATH . '/includes/class-wc-payments.php';
+	require_once WCPAY_ABSPATH . '/src/wcpay-get-container.php';
```

Comment:
Interesting suggestion. I added it in https://github.com/Automattic/woocommerce-payments/pull/6862/commits/e7e7842549f6b8bb6478deb312f93e17cb52c34b, let's see how it works.

What concerns me a bit is that this way we have less control over the order of what's loaded. On the other hand, nothing should happen there, besides registering a function, so we should be good to go.

It will be interesting what happens once more stuff gets added to `src`, and some of it needs to be loaded directly, instead of through a method in `includes`. In the worst case we'll revert this change, but I have a suspicion that it might end up being helpful instead, forcing us into a more elegant solution.